_episodes/01-connecting.md

```diff
@@ -129,6 +128,93 @@ For those logging in with PuTTY it would likely be best to cover the terminal
 basics already mentioned above before moving on to navigating the remote
 system.
 
+## Creating an SSH key
+
+SSH keys are an alternative method for authentication to obtain access to 
+remote computing systems. They can also be used for authentication when
+transferring files or for accessing version control systems. In this section
+you will create a pair of SSH keys, a private key which you keep on your
+own computer and a public key which is placed on the remote HPC system
+that you will log in to.
+
+### Linux, Mac and Windows Subsystem for Linux
+
+Once you have opened a terminal check for existing SSH keys and filenames
+since existing SSH keys are overwritten,
+```
+$ ls ~/.ssh/
+```
+{: .language-bash}
+
+then generate a new public-private key pair,
+```
+$ ssh-keygen -t ed25519 -a 100 -f ~/.ssh/id_{{ site.workshop_host }}_ed25519
+```
+{: .language-bash}
+
+- `-o` (no default): use the OpenSSH key format,
+  rather than PEM.
+- `-a` (default is 16): number of rounds of passphrase derivation;
+  increase to slow down brute force attacks.
+- `-t` (default is [rsa](https://en.wikipedia.org/wiki/RSA_(cryptosystem))):
+  specify the "type" or cryptographic algorithm. 
+  [ed25519](https://en.wikipedia.org/wiki/EdDSA)
+ is faster and shorter than RSA for comparable strength.
+- `-f` (default is /home/user/.ssh/id_algorithm): filename to store your keys.
+  If you already have SSH keys, make sure you specify a different name:
+  `ssh-keygen` will overwrite the default key if you don't specify!
+
+If EcDSA is not available, use the older (but strong and trusted)
```

Comment:
ED25519 is EdDSA (Edwards-curve), not EcDSA (elliptic-curve), though the two are related.

Reply:
updated to use ed25519 instead of EcDSA